.gitattributes

```diff
@@ -0,0 +1 @@
+vendor/**/* linguist-generated=true
```

Comment:
`vendor/` files still show up in diff stats 🤔

Reply:
I've updated the `.gitattributes` file with

```
vendor/** linguist-vendored
go.sum linguist-generated
```

But the files still seem to be visible 🤔

Reply:
Let's leave it as is.